### ISSUE TITLE
[Bug]: Metamask recommended gas fees for Cyber Network is too high

### ISSUE THREAD

**Issue by PengDeng-Cyber** [MetaMask/metamask-extension#24236]
### Describe the bug

Metamask suggests a very high gas fee on Cyber Network. Maybe Metamask had set a default minimum recommendation price for gas based on the average of all networks.

Cyber Network is a fully EVM-compatible Rollup built through the OP Stack. And here are some infos for Cyber testnet and mainnet.

```
Mainnet Info:
Chain ID: 7560
RPC: https://cyber.alt.technology/
Explorer: https://cyberscan.co

Testnet Info:
Chain ID: 111557560
RPC: https://cyber-testnet.alt.technology/
Explorer: https://testnet.cyberscan.co
```

And here is the recommended gas fee by Metamask.
![image](https://github.com/MetaMask/metamask-extension/assets/104816653/2f997825-3f3a-439d-9d9e-40edf94946b7)
The recommended gas price is default 1.5gwei.
And here is the real gas price on Cyber mainnet.
```
curl --location 'https://cyber.alt.technology/' \
--header 'Content-Type: application/json' \
--data '{"method":"eth_maxPriorityFeePerGas","params":[],"id":1,"jsonrpc":"2.0"}'
{"jsonrpc":"2.0","id":1,"result":"0xf4240"}

cast gas-price -r=https://cyber.alt.technology/
1000252
```
The real gas price is much lower than the default minimum recommendation price, it is 1000252wei (0.001000252gwei).

### Expected behavior

We expect that Metamask can provide a recommended gas fee based on the real gas price on Cyber chain.

### Steps to reproduce

1. Add custom network using Cyber network info.
```
Mainnet Info:
Chain ID: 7560
RPC: https://cyber.alt.technology/
Explorer: https://cyberscan.co

Testnet Info:
Chain ID: 111557560
RPC: https://cyber-testnet.alt.technology/
Explorer: https://testnet.cyberscan.co
```
2. Send asset on Metamask, edit "Estimated fee".
3. Click on "Advanced", and can see the Priority Fee is default 1.5gwei, the Max base fee is 1.500000302gwei.
4. Get the real gas price from Cyber chain rpc.
```
curl --location 'https://cyber.alt.technology/' \
--header 'Content-Type: application/json' \
--data '{"method":"eth_maxPriorityFeePerGas","params":[],"id":1,"jsonrpc":"2.0"}'
{"jsonrpc":"2.0","id":1,"result":"0xf4240"}

cast gas-price -r=https://cyber.alt.technology/
1000252
```
The Cyber gas price is 0.001000252gwei.

### Version

11.13.1

### Build type

None

### Browser

Chrome

### Operating system

MacOS

**Comment by gauthierpetetin** [comment]
Hi @MetaMask/confirmations-system-team , this issue seems similar to this one: https://github.com/MetaMask/metamask-extension/issues/24120

**Comment by bschorchit** [comment]
Hey @PengDeng-Cyber, sorry for the delay on replying to you. Closing as this should be fixed in the latest releases (starting on v11.16 release).
Let us know if that is not the case!
